lib/clerk/authenticatable.rb

```diff
@@ -1,97 +1,79 @@
-require "active_support/concern"
+# frozen_string_literal: true
+
+require 'active_support/concern'
 
 module Clerk
+  # Concern containing Clerk Auth related helper methods
   module Authenticatable
     extend ActiveSupport::Concern
 
     protected
 
-    # Makes a request to the Clerk API to verify the session again and return
-    # the Session object. Subsequent calls to this method will return the cached
-    # Session object.
-    #
-    # NOTE: For better performance, you can instead use `#clerk_verified_session_claims`
-    # which already contains the verified claims as retrieved from the session
-    # token.
+    # Returns the verified claims as retrieved from the session token.
     def clerk_session
```

Comment:
:question: Shouldn't we name this `session_claims`? 

`session` usually refers to the Session resource.

Reply:
My 2c: I think I agree with `session_claims` as a name, or perhaps use a uniform naming convention with JS & Go SDKs.